cocos/core/renderer/scene/model.ts

```diff
@@ -375,10 +376,10 @@ export class Model {
     protected _localBuffer: Buffer | null = null;
 
     /**
-     * @en Instance matrix id
-     * @zh 实例矩阵索引
+     * @en Instance matrix id, access by subModel
+     * @zh 硬件实例化中的矩阵索引，通过子模型访问
      */
-    private _instMatWorldIdx = -1;
+    private _instMatWorldIdx: Map<SubModel, number> = new Map<SubModel, number>();
```

Comment:
Not it is a map, so i think should rename the variable too, such as `_instanedMatrixWorldIndexMap`.